try:
    number1 = 100
except Exception as e:
    print(e)